deep_field_metadetect/jaxify/tests/test_jax_deep_metacal.py

```diff
@@ -0,0 +1,339 @@
+    noise_fac = 1 / np.sqrt(10)
+    nsims = nchunks * chunk_size
+
+    rng = np.random.RandomState(seed=34132)
+    seeds = rng.randint(size=nsims, low=1, high=2**29)
+    res_p = []
+    res_m = []
+    loc = 0
+    for chunk in range(nchunks):
+        _seeds = seeds[loc : loc + chunk_size]
+        # jobs = [
+        #     joblib.delayed(_run_sim_pair)(
+        #         seed, s2n, noise_fac, 0.8, skip_wide, skip_deep
+        #     )
+        #     for seed in _seeds
+        # ]
+        # outputs = joblib.Parallel(n_jobs=-1, verbose=10)(jobs)
+        for seed in _seeds:
+            res = _run_sim_pair(seed, s2n, noise_fac, 0.8, skip_wide, skip_deep)
+            if res is not None:
+                res_p.append(res[0])
+                res_m.append(res[1])
+
+        if len(res_p) < 500:
+            njack = len(res_p)
+        else:
+            njack = 100
+
+        m, merr, c1, c1err, c2, c2err = estimate_m_and_c(
+            np.concatenate(res_p),
+            np.concatenate(res_m),
+            0.02,
+            jackknife=njack,
+        )
+
+        print("# of sims:", len(res_p), flush=True)
+        print_m_c(m, merr, c1, c1err, c2, c2err)
+
+        if not skip_wide and not skip_deep:
+            assert np.abs(m) < max(MAX_ABS_M, 3 * merr), (m, merr)
+        elif 3 * merr < 5e-3:
+            assert np.abs(m) >= max(MAX_ABS_M, 3 * merr), (m, merr)
+            # if we are more than 10 sigma biased, then the test
+            # has passed for sure
+            if np.abs(m) / max(MAX_ABS_M / 3, merr) >= 10:
+                break
+        assert np.abs(c1) < max(4.0 * c1err, MAX_ABS_C), (c1, c1err)
+        assert np.abs(c2) < max(4.0 * c2err, MAX_ABS_C), (c2, c2err)
+
+        loc += chunk_size
+
+    print_m_c(m, merr, c1, c1err, c2, c2err)
+    if not skip_wide and not skip_deep:
+        assert np.abs(m) < max(MAX_ABS_M, 3 * merr), (m, merr)
+    else:
+        assert np.abs(m) >= max(MAX_ABS_M, 3 * merr), (m, merr)
+    assert np.abs(c1) < max(4.0 * c1err, MAX_ABS_C), (c1, c1err)
+    assert np.abs(c2) < max(4.0 * c2err, MAX_ABS_C), (c2, c2err)
+
+
+def _run_single_sim_maybe_mcal(
+    seed,
+    s2n,
+    g1,
+    g2,
+    deep_noise_fac,
+    deep_psf_fac,
+    use_mcal,
+    zero_flux,
+):
+    obs_w, obs_d, obs_dn = make_simple_sim(
+        seed=seed,
+        g1=g1,
+        g2=g2,
+        s2n=s2n,
+        deep_noise_fac=deep_noise_fac,
+        deep_psf_fac=deep_psf_fac,
+        obj_flux_factor=0.0 if zero_flux else 1.0,
+        return_NT=True,
+    )
+    if use_mcal:
+        mcal_res = jax_metacal_op_shears(
+            obs_w,
+            dk=jnp.pi / (53 * 0.2) / 4,
+        )
+        for key, value in mcal_res.items():
+            mcal_res[key] = NT_to_ngmix_obs(value)
+    else:
+        mcal_res = jax_metacal_wide_and_deep_psf_matched(
+            obs_w,
+            obs_d,
+            obs_dn,
+            dk_w=jnp.pi / (53 * 0.2) / 4,
+            dk_d=jnp.pi / (53 * 0.2) / 4,
```

Comment:
These factors for dk don't have the factor of `2` included above. Why do they differ?